src/client/components/app/controller.ts

```diff
@@ -0,0 +1,15 @@
+import { action } from 'mobx'
+import { RobotModel } from '../robot/model'
+
+export class AppController {
+
```

Comment:
Looks like we want http://eslint.org/docs/rules/padded-blocks

Unfortunately `tslint` nor `tslint-eslint-rules` supports it 😞